cfn_encrypt/ssm_parameter.py

```diff
@@ -19,16 +21,45 @@ def handler(event, context):
     ch = logging.StreamHandler()
     logger.addHandler(ch)
     name = event["ResourceProperties"]["Name"]
-    try:
+    value = None
 
+    try:
         if event["RequestType"] in ["Create", "Update"]:
             if event["RequestType"] == "Create" and parameter_exist(name):
                 raise NameError("A Parameter named {} already exists".format(name))
 
+            generate_password = event["ResourceProperties"]["GeneratePassword"] if "GeneratePassword" in event["ResourceProperties"] else None
+            value = event["ResourceProperties"]["Value"] if "Value" in event["ResourceProperties"] else None
+
+            if value and generate_password in ['true', 'True', '1', True, 1]:
```

Comment:
Here it only needs to check if both vars are set right?

Reply:
Valid point, but this is to prevent the use-case that the message is shown when a value is filled in and the GeneratePassword Property has been set to false.